simplesat/constraints/requirement.py

```diff
@@ -47,6 +48,41 @@ class Requirement(object):
     specs: seq
         Sequence of constraints
     """
+
+    @classmethod
+    def from_constraints(cls, constraint_tuple):
+        """ Return a Requirement object from a PackageMetadata constraint
+        tuple.
+
+        Parameters
+        ----------
+        constraints : constraints tuple
+            A tuple of constraints like (
+                'nose', ( # disjuncitons
```

Comment:
What's written here is backwards actually. They get sorted so it should be `('< 1.4', '>= 1.3')`.